res_partner_fiscal_position/__openerp__.py

```diff
@@ -0,0 +1,35 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
```

Comment:
This line can simply be deleted. It' s not OpenERP anymore but odoo, but this is a separate addon, so need for this line.